client/src/pages/Suggest/index.jsx

```diff
@@ -41,11 +45,13 @@ export default function Suggest() {
 
 	return (
 		<>
-			<h2>Suggest a resource</h2>
+			<h2>{isAdmin ? "Publish a resource" : "Suggest a resource"}</h2>
 			<p>
-				Please use the form below to submit a suggestion. Note that it will not
+				{isAdmin
+					? "Please use the form below to submit a resource. It's will appear immediately to the home page because you are administrator"
```

Comment:
💡 Typos in UI text
```suggestion
					? "Please use the form below to submit a resource. It will appear immediately on the home page because you are administrator."
```